lib/dw/dataset/index.mjs

```diff
@@ -37,6 +37,19 @@ export default function Dataset(columns) {
         }
     }
 
+    function escapeCSVValue(value, delimiter, quoteChar) {
```

Comment:
Maybe add some comments to explain what this function returns? Particularly the first return, which may look pretty wild to somebody not familiar with escaping in CSV. You could also add a link to one of the [CSV specs](https://csv-spec.org/#csv-format-specification).